game-backend/src/helper/gameObjects.test.ts

```diff
@@ -73,4 +75,22 @@ describe('getGameObjectsPerType', () => {
       expect(foods[i].type).toBe(GAME_OBJECT_TYPES.FOOD);
     }
   });
+
+  it('getFreeFields should filter out occupied fields', () => {
+    const state: State = StateBuilder.buildState()
+      .addUnit({ id: 'UNIT_ID', x: 1, y: 1 })
+      .addFood({ id: 'FOOD_1_ID', x: 2, y: 2 })
+      .build();
+
+    const freeFields = getFreeFields(state);
+
+    expect(freeFields.length).toBe(ARENA_HEIGHT * ARENA_WIDTH - 2);
+    expect(freeFields).not.toEqual(
+      expect.arrayContaining([
+        { x: 1, y: 1 },
+        { x: 2, y: 2 },
+      ])
+    );
+    expect(freeFields).toEqual(expect.arrayContaining([{ x: 1, y: 2 }]));
+  });
```

Comment:
Shouldn't this test be in its own `describe {}` block because the tested function is `getFreeFields` not `getGameObjectsPerType`?